src/Features/Core/Completion/Snippets/ISnippetCompletionListProvider.cs

```diff
@@ -0,0 +1,8 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Completion.Snippets
+{
+    internal interface ISnippetCompletionListProvider
```

Comment:
This is just a temporary measure. It's a marker interface so we can hide some of the ugliness around snippets right now. Hopefully I'll be able to just remove this later. I'll add a comment to that effect.